skrf/frequency.py

```diff
@@ -580,14 +580,34 @@ def copy(self) -> Frequency:
         freq.unit = self.unit
         return freq
 
+    def _t_padded(self, *, pad: int = 0, n: int | None = None, bandpass: bool | None = None) -> np.ndarray:
```

Comment:
This is probably more clear than the `fftfreq` trick. Also it is a good idea to have all the time vector made in frequency.py so we are sure it is the same as for time gating.